docs/USAGE.md

```diff
@@ -42,102 +46,124 @@ pip install -e .
+3. Train models using cyto_dl. Ensure to run the training scripts from the folder where the repo was cloned (and where all the data was downloaded). [Experiment configs](../configs/experiment/) for point cloud and image models for the cellpack dataset are located here:
 
 ```
-└── configs
-    └── experiment
-        └── cellpack
-            ├── image_classical.yaml <- Classical image model experiment
-            ├── image_so3.yaml <- Rotation invariant image model experiment
-            └── pc_classical.yaml    <- Classical point cloud model experiment
-            └── pc_so3.yaml    <- Rotation invariant point cloud model experiment
+configs
+└── experiment
+    └── cellpack
+        ├── image_classical.yaml <- Classical image model experiment
+        ├── image_so3.yaml       <- Rotation invariant image model experiment
+        ├── pc_classical.yaml    <- Classical point cloud model experiment
+        └── pc_so3.yaml          <- Rotation invariant point cloud model experiment
 ```
 
-Here is an example of training a rotation invariant point cloud model
+Here is an example of training a rotation invariant point cloud model.
 
 ```bash
 python src/br/models/train.py experiment=cellpack/pc_so3
 ```
 
-Override parts of the experiment config via command line or manually in the configs. For example, to train a classical model, run
+Override parts of the experiment config via command line or manually in the configs. For example, to train a classical model, run the following.
 
 ```bash
 python src/br/models/train.py experiment=cellpack/pc_so3 model=pc/classical_earthmovers_sphere ++csv.save_dir=[SAVE_DIR]
 ```
 
-4. To compute embeddings from the trained models, update the data paths in the [datamodule files](../configs/data/) and run
+# 2. Model inference
 
-```
-python src/br/analysis/run_embeddings.py --save_path "./outputs/" --sdf False --dataset_name "pcna" --batch_size 5 --debug False
-```
+## Steps to download pre-trained models
+
+To skip model training, download pre-trained models. For each of the six datasets, there are five `.ckpt` files to download.
+
+| Download `.ckpt` files from here                                                                                                                                                                         | To this folder                               |
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| [cellPACK synthetic dataset](https://open.quiltdata.com/b/allencell/tree/aics/morphology_appropriate_representation_learning/model_checkpoints/cellpack/)                                                | `configs/experiment/cellpack/ckpts`          |
```

Comment:
The "To this folder" does not need to be so strict

Reply:
Removed the table entirely: [f416574](https://github.com/AllenCell/benchmarking_representations/pull/61/commits/f416574e47f28ec38bfa09206abcdc04a6d426ca)